appcelerator-archive/amp
# update haproxy to version 1.0.3

Update Haproxy version to 1.0.3 (previous was 1.0.2)
It's a clean version merging fixes from 1.0.1 and 1.0.2, no new features regarding 1.0.2

test:
- make install
- make TAG=local build-image
- ampadm pf pull -v --local
- ampadm pf start -v --local
- amp stack up test -f ./api/rpc/stack/test_samples/sample-03.yml
- curl www.test.local.axwaytest.net/ping
- return should be: pong
- amp stack rm test -f
